src/lib/genesis_ledger/intf.ml

```diff
@@ -1,11 +1,31 @@
 open Coda_base
 open Signature_lib
+open Core_kernel
+
+module Timing = struct
+  type t =
+    { initial_minimum_balance: int
+    ; cliff_time: int (*slot number*)
+    ; vesting_period: int (*in slots*)
+    ; vesting_increment: int }
+
+  let gen =
+    let open Quickcheck.Generator.Let_syntax in
+    let%bind initial_minimum_balance =
+      Quickcheck.Generator.small_non_negative_int
+    in
+    let%bind cliff_time = Quickcheck.Generator.small_non_negative_int in
+    let%bind vesting_increment = Quickcheck.Generator.small_non_negative_int in
+    let%map vesting_period = Quickcheck.Generator.small_positive_int in
+    {initial_minimum_balance; cliff_time; vesting_increment; vesting_period}
+end
 
 module Public_accounts = struct
   type account_data =
     { pk: Public_key.Compressed.t
     ; balance: int
-    ; delegate: Public_key.Compressed.t option }
+    ; delegate: Public_key.Compressed.t option
+    ; timing: Timing.t option }
```

Comment:
in `Coda_base.Account`, the timing is either `Timed _` or `Untimed`, rather than an option.

Would it be better to follow that discipline here, or else make it an option in `Account`?

Reply:
Did you want me to change it to `Timed _ | Untimed` only here or even in `runtime_config.ml`? Using `option` seemed cleaner in `Runtime_config`

Reply:
If you think it's cleaner there, go with that.